default_metadata/component/checkbox.json

```diff
@@ -0,0 +1,9 @@
+{
+  "_id": "component.checkbox",
+  "_type": "checkbox",
+  "errors": {},
+  "hint": "Component hint",
+  "items": [],
+  "name": "component-name",
+  "legend": "Required legend"
+}
```

Comment:
so this file will need to be `checkboxes.json`

and in that case we're missing a `default_metadata/definition/checkbox.json`

Reply:
also you're default metadata for the checkboxes should add two items by default. similar to the radios.json

https://github.com/ministryofjustice/fb-metadata-presenter/blob/main/default_metadata/component/radios.json